hyperbee/demo/src/dbPut.mjs

```diff
@@ -9,10 +9,18 @@ import { makeKey } from "./makeKey.mjs";
 
 // TODO: load from elsewhere.
 // DO NOT USE IN PRODUCTION
-const privKey = ed.utils.randomPrivateKey();
+const privKey = Buffer.from(
```

Comment:
We might want to use Uint8Array if we want similar code in the future to work in browsers as well. Buffer is Node only, but is a subclass of Uint8Array.